src/actions/enter.ts

```diff
@@ -105,7 +106,7 @@ export class EnterAction {
       const block = curNode.getLocation() as Block;
       if (!this.tryShowFullBlockFieldEditor(block)) {
         const shortcut = getShortActionShortcut('list_shortcuts');
-        const message = `Press ${shortcut} for help on keyboard controls`;
+        const message = Msg['HELP_PROMPT'].replace('%1', shortcut);
```

Comment:
Here & elsewhere: how descriptive do the names need to be? If they are global context, I'd expect something like: `KEYBOARD_SHORTCUTS_HELP_PROMPT` to be much clearer and avoid potential duplications in the future. If this level of specificity is preferred, can all of the other ones be updated accordingly?

Reply:
...I realize we just release core, so we're kind of committed now. Sorry I missed this until I went looking for things to pick up now that that was out the door :/